lib/manager/components/GeneralSettings.js

```diff
@@ -155,7 +156,8 @@ export default class GeneralSettings extends Component<Props, State> {
             </ListGroupItem>
           </ListGroup>
         </Panel>
-        <Panel header={<h3>Automatic fetch</h3>}>
+        <Panel>
+          <Panel.Heading>Automatic Fetch</Panel.Heading>
```

Comment:
The `AutoPublishSettings` pane also needs to be converted to this version of react-bootstrap.

Reply:
Converted in 5353f432defd32577a1378e0857c2946989d4bcd